scripts/hracq.py

```diff
@@ -88,6 +88,9 @@ def set_hardware_settings(self, wavelength, power=None):
         """
         if power is None:
             power = self.light.power.range[-1]
+        elif type(power) == float:
+            # Convert the power commandline argument float value into a list (to match light power type)
+            power = [power]
```

Comment:
Does this make sense? @pieleric

Reply:
Good catch, `power` should indeed be a number. That said, to check for a number, it's better to write something like: if `isinstance(power, numbers.Number)`. This catches integers too.